From d8ba58bec4812a818d8f336b3c0bbeec325fc46f Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Sun, 7 Oct 2018 11:02:16 +0800
Subject: [PATCH] arm64: dts: rockchip: fix irq flags of fiq-debugger for
 rk1808-evb

irq trigger level flags only support HIGH for GIC.

Fix this warning:
  genirq: Setting trigger mode 8 for irq 188 failed (gic_set_type+0x0/0x74)

Change-Id: I3ef2bb7d26003595b8c6dafb932512a0f5eae450
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index b782eefd98c2..7ec7e1b79478 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -94,7 +94,7 @@
 		/* If enable uart uses irq instead of fiq */
 		rockchip,irq-mode-enable = <0>;
 		rockchip,baudrate = <1500000>;  /* Only 115200 and 1500000 */
-		interrupts = <GIC_SPI 242 IRQ_TYPE_LEVEL_LOW>;
+		interrupts = <GIC_SPI 242 IRQ_TYPE_LEVEL_HIGH>;
 		status = "okay";
 	};
 
-- 
2.35.3

